src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -144,20 +144,3 @@ CREATE TABLE warehouse (
 --add constraints and indexes
 CREATE INDEX idx_customer_name ON customer (c_w_id,c_d_id,c_last,c_first);
```

Comment:
Change this to `((c_w_id, c_d_id) HASH, c_last, c_first)`

Reply:
Done.